package-lock.json

```diff
@@ -0,0 +1,2850 @@
+{
```

Comment:
Atatata... Remember what we've talked about regading the package-lock?

Let's remove the file and add it to .gitignore for future convinience.

Reply:
https://git-scm.com/docs/gitignore - that's what the gitignore is. It's usually used for things you want to have in the project directory, but don't want to commit. Normally that would be database files, dependencies, configs etc.